src/components/Premium/NewPopularList.tsx

```diff
@@ -0,0 +1,171 @@
+import styled from 'styled-components';
+
+import { 더바디샵티트리오일, 록시땅로즈오드뚜왈렛, 비오템옴드모이스춰라이저, 비오템옴드젤로션, 오리진스앰플크림, 헤라파우더 } from '../../assets/image';
+import { ProductData } from '../../types/common';
+
+import NewPopularProduct from './NewPopularProduct';
+
+const NewPopularList = () => {
+
+  const ProductData: ProductData[] = [
+    {
+      name : '오리진스 MAD 플러스 리주버네이팅 수분 앰플 크림50ml',
+      originalPrice: 54000,
+      discountRate: 20,
+      discountPrice: 43200,
+      image: 오리진스앰플크림,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: true,
+        증정: true,
+        '1+1': false,
+      },
+    },
+    {
+      name : '록시땅 로즈 오 드 뚜왈렛 75ml (2022)',
+      originalPrice: 70000,
+      discountRate: 10,
+      discountPrice: 71100,
+      image: 록시땅로즈오드뚜왈렛,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: false,
+        증정: false,
+        '1+1': false,
+      },
+    },
+    {
+      name : '헤라 소프트 피니시 루스 파우더 15g',
+      originalPrice: 60000,
+      discountRate: 0,
+      discountPrice: 60000,
+      image: 헤라파우더,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: true,
+        증정: false,
+        '1+1': false,
+      },
+    },
+    {
+      name : '더비다샵 티트리 오일 10ml',
+      originalPrice:14000,
+      discountRate: 5,
+      discountPrice: 13300,
+      image: 더바디샵티트리오일,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: true,
+        증정: false,
+        '1+1': false,
+      },
+    },
+    {
+      name : '[박보검 PICK] 비오템옴드 포스수프림 모이스춰라이저 50ml',
+      originalPrice: 140000,
+      discountRate: 0,
+      discountPrice: 140000,
+      image: 비오템옴드모이스춰라이저,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: true,
+        증정: false,
+        '1+1': false,
+      },
+    },
+    {
+      name : '비오템옴드 포스수프림 어드밴스드 젤 로션 75ml',
+      originalPrice: 90000,
+      discountRate: 0,
+      discountPrice: 90000,
+      image: 비오템옴드젤로션,
+      likeTF: false,
+      tags: {
+        BEST: false,
+        단독: false,
+        오늘드림: true,
+        증정: true,
+        '1+1': false,
+      },
+    },
+  ];
+
+  const ProductList = ProductData.map((item) => {
+    return <NewPopularProduct key = {item.name} productData = {item} />;
+  });
+
+  return (
+    <St.NewPopularContainer>
+      <St.Header>
+        <div className = "new"> 신상품 </div>
+        <div className = "popular"> 인기상품 </div>
+      </St.Header>
+      <St.ProductList>
+        {ProductList}
+      </St.ProductList>
+    </St.NewPopularContainer>
+  );
+};
+
+export default NewPopularList;
+
+const St = {
+  NewPopularContainer : styled.section`
+  display: flex;
+  flex-direction: column;
+
+  width: 37.5rem;
```

Comment:
그그. . 전체적으로 width를 37.5rem으로 고정해놓는거보단 width:100% 또는 100vw를 사용하는 것이 어떄??
프리미엄 페이지 보면 헤더랑 네브바, 프리미엄 브랜드 부분은 width:100vw로 되어있어서 화면 너비 늘렸을 때 같이늘어나거든 이렇게
<img width="558" alt="스크린샷 2023-05-24 오전 12 01 53" src="https://github.com/GOSOPT-CDS-TEAM2/frontend/assets/55528304/522ccc89-55ec-45d2-b084-2e73564a8c0e">

근데 이 아래부터는 width가 37.5로 고정되어있더라구!! 

전에 디쟌분들께 물어봤을때 `요소의 너비나 요소사이 간격을 늘리고 양쪽 여백을 고정하기`로 합의했어서 훔
사실 중요한건 아니니까 후순위로 미뤄도 될거같구 나중에 여유있으면 우리끼리 맞춰야할거같긴해! 

고친다면 이렇게 하면 될거같아!!

```suggestion
  flex-direction: column;
  align-items: center;
  
  width: 100%;
```